tardis/montecarlo/base.py

```diff
@@ -0,0 +1,31 @@
+from tardis.montecarlo import montecarlo
+
+class MontecarloRunner():
```

Comment:
Ok, thanks! You don't happen to see any glaring errors that might be causing my `transition_probabilities` issue, do you? :(